a11y-dialog.js

```diff
@@ -399,4 +399,4 @@
   } else if (typeof global === 'object') {
     global.A11yDialog = A11yDialog;
   }
-}(this));
+}(typeof global !== "undefined" ? global : window));
```

Comment:
Actually, that's a perfectly reasonable thing to ask. In fact, I'm kind of embarrassed (puzzled, even) that I hadn't noticed this myself.

Either way, thank you!